_health-care/_js/components/financial-assessment/ChildInformationSection.jsx

```diff
@@ -49,7 +51,10 @@ class ChildInformationSection extends React.Component {
       );
     }
 
-    if (this.props.data.hasChildrenToReport === true) {
+    // TODO: Yes/No is a weird case for the related radio buttons, this should probably be boolean.
+    //        Should we make a specific case for this?
+    if (this.props.data.hasChildrenToReport === true ||
+        this.props.data.hasChildrenToReport === 'Yes') {
```

Comment:
We should only have one possible stored value, not two. The reason why we were checking for `true` before was because it was a checkbox. We could have the labels for the radio buttons be `Yes` and `No` and still store the value as `true` and `false`. Let's follow whatever the original form does in terms of the value and stick with that.